pkg/cli/runner.go

```diff
@@ -156,14 +159,22 @@ Options:
 		}
 	}
 
+	// https://github.com/suzuki-shunsuke/github-comment/issues/1489
+	if flags.GHEBaseURL == "" {
+		flags.GHEBaseURL = os.Getenv("GITHUB_API_URL")
+	}
```

Comment:
The `GITHUB_API_URL` has `https://<hostname>/api/v3` style, but auto injected /api/v3 suffix when without URL path (and trailing slash) by WithEnterpriseURLs() method. 

So, users can use both style:

- GITHUB_API_URL=https://github.example.jp, https://github.example.jp/
- GITHUB_API_URL=https://github.example.jp/api/v3, https://github.example.jp/api/v3/